sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateTableAsSelect.scala

```diff
@@ -91,6 +93,6 @@ case class CreateTableAsSelect(
   }
 
   override def argString: String = {
-    s"[Database:$database, TableName: $tableName, InsertIntoHiveTable]\n" + query.toString
+    s"[Database:$database, TableName: $tableName, InsertIntoHiveTable]"
```

Comment:
But `tableDesc` will be helpful if user want to get more information on what backend for the created table. 
Like will it be parquet format by default in spark sql, which is not the same as hive does.